open-sphere-plugins/open-sensor-hub/src/main/java/io/opensphere/osh/mantle/OSHMantleController.java

```diff
@@ -121,13 +121,16 @@ public boolean addServer(String serverName, String baseUrl)
         boolean success = false;
         try
         {
-            List<Offering> offerings = myQuerier.getCapabilities(baseUrl);
-            addDynamicService(serverName, createServerGroupService(serverName, baseUrl, offerings));
-            success = true;
+            List<Offering> offerings = myQuerier.getCapabilities(serverName, baseUrl);
+            if (!offerings.isEmpty())
+            {
+                addDynamicService(serverName, createServerGroupService(serverName, baseUrl, offerings));
+                success = true;
+            }
         }
         catch (QueryException e)
         {
-            LOGGER.error(e);
+            LOGGER.error(e.getCause().getMessage());

```

Comment:
I would think we would want to still log the exception and not just the message.